spec/tests/spec/PhpSpec/TestWithArrayAccessSpec.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace spec\tests\spec\PhpSpec;
+
+use PhpSpec\ObjectBehavior;
+use tests\spec\PhpSpec\TestWithArrayAccess;
+
+class TestWithArrayAccessSpec extends ObjectBehavior
```

Comment:
Specs are for testing the classes in the codebase

Please move this test into a feature file, and if possible add some tests that use `$this['foo']`-type access to check we didn't break it

Reply:
Fixed.